BioMed.Api/BioMed.Services/Services/AnalysisTypeService.cs

```diff
@@ -6,19 +6,23 @@
 using BioMed.Domain.Pagination;
 using BioMed.Domain.ResourceParameters;
 using BioMed.Infrastructure.Persistence;
+using Microsoft.Extensions.Logging;
 
 namespace BioMed.Services.Services
 {
     public class AnalysisTypeService : IAnalysisTypeService
     {
         private readonly IMapper _mapper;
         private readonly BioMedDbContext _context;
+        private readonly ILogger<AnalysisTypeService> _logger;
```

Comment:
Why do you write Logger, but don`t use it?

Reply:
Yes, I forgot to remove, I will fix it